tests/storage/storage.py

```diff
@@ -70,3 +70,13 @@ def get_xapi_session():
         'vdiUuid': vdi.uuid,
         'srRef': master.execute_script(get_sr_ref, shebang='python')
     }))
+
+def run_quicktest_on_sr(sr):
```

Comment:
Maybe this could be a method of SR

Reply:
I confirm that I run it with
```
sr.pool.master.ssh(['/opt/xensource/debug/quicktest', '-sr', sr.uuid])
```
So no need to have `get_default_sr` and `set_default_sr` anymore (at least when `-sr` will be available)

Reply:
IMHO this should stay as test method as it's only used in the `storage` scope.

Reply:
A method this simple and whose main input is a SR UUID, I would have put it on the SR class anyway (whose scope is storage anyway), and avoided a lot of imports in test files.